common/static/js/spec/utility_spec.js

```diff
@@ -16,3 +16,17 @@ describe('utility.rewriteStaticLinks', function() {
         ).toBe('<img src="http://www.mysite.org/static/foo.x"/>');
     });
 });
+describe('utility.rewriteCdnLinksToStatic', function() {
+    'use strict';
+    it('does not replace of "cdn url" to "static url" if url is not absolute url', function() {
+        expect(
+            rewriteCdnLinksToStatic('<img src="/assets/foo.x"/>') // eslint-disable-line no-undef
+        ).toBe('<img src="/assets/foo.x"/>');
+    });
+    it('does a replace of "cdn url" to "static url" if url is part of absolute url', function() {
```

Comment:
Seems like `rewriteCdnLinksToStatic` is also capable of replacing CDN URL which either starts with `https` and `www`
e.g. `https://prod-edxapp.edx-cdn.org/assets/foo.x`
`//www.prod-edxapp.edx-cdn.org/assets/foo.x`
`www.prod-edxapp.edx-cdn.org/assets/foo.x`

Reply:
> https://prod-edxapp.edx-cdn.org/assets/foo.x
> //www.prod-edxapp.edx-cdn.org/assets/foo.x
The above-mentioned URLs can be replaced. The URL "www.prod-edxapp.edx-cdn.org/assets/foo.x" can not be replaced as it doesn't match the pattern in the given method "rewriteCdnLinksToStatic".